backend/app/auth/supabase/supabase_auth_exception.py

```diff
@@ -0,0 +1,19 @@
+from dataclasses import dataclass
+
+from supabase import AuthError
+
+from app.auth import AuthException
+from app.models.types import NonBlankString
+
+
+@dataclass(frozen=True)
+class SupabaseAuthException(AuthException):
```

Comment:
Just use the raise AuthException from original_exception syntax instead of this class.

Reply:
I am using this syntax now.